### PR TITLE
Issue 54 fix good

### PR DESCRIPTION
**Pull request by drouba**
fix for issue #54 
assigning instructors to lectures should not be based on ID and the dropdown should contain more description

### CODE REVIEW

File: public/instructors_list.php [hunk 1]
```diff
@@ -153,9 +163,9 @@
         Course Lecture: 
         <select Name="lecture_selection" id="lecture_selection">
             <option value="">----Select----</option>
-        <?php foreach($lecture as $row) { ?>
-            <option value="<?php echo $row->id; ?>">
-            <?php echo $row->id; ?>
+        <?php foreach($unassigned_lectures as $row) { ?>
+            <option value="<?php echo $row->lecture_code; ?>">
```
Comment (Zachary-Concordia) [inline public/instructors_list.php:167]: I think this should be `<option value="<?php echo $row->id; ?>"`? The lecture table doesn't have a lecture_id field.

**Review by Zachary-Concordia — CHANGES_REQUESTED** [review]
Doesn't assign professors to sections on submit, throws error regarding the id. See comment on L167

File: public/instructors_list.php [hunk 2]
```diff
@@ -25,6 +25,16 @@
     echo "<br>" . $error->getMessage();
 }
 
+$assigned_lectures = array(); 
+
+foreach($course_assignment as $assigned){
+    array_push($assigned_lectures, $assigned->lecture);
+} 
+
```
[the remark this answers is not in the document]
Comment (jamestjw) [reply public/instructors_list.php:33]: you should use the lambda function notation for the array_udiff too

**Review by jamestjw — CHANGES_REQUESTED** [review]

File: public/instructors_list.php (hunk 1 above)
Comment (drouba) [reply public/instructors_list.php]: done

File: public/instructors_list.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (drouba) [reply public/instructors_list.php]: done

File: public/instructors_list.php [hunk 3]
```diff
@@ -93,7 +97,7 @@
 
 <?php if (isset($_POST['submit']) && $create_success) { ?>
     <blockquote><?php echo $_POST['first_name']; ?> successfully added.</blockquote>
-    <?php header("refresh: 1")?>
+    <?php //header("refresh: 1")?>
```
Comment (jamestjw) [inline public/instructors_list.php:100]: is this intended?

**Review by jamestjw — APPROVED** [review]
ok all good just look at the latest comment I left

**Review by Zachary-Concordia — APPROVED** [review]

File: public/instructors_list.php (hunk 3 above)
Comment (drouba) [reply public/instructors_list.php:100]: oups no, that was for testing purposes

**Pull request merged**